certbot/renewal.py

```diff
@@ -408,6 +409,16 @@ def handle_renewal_request(config):
         try:
             if renewal_candidate is None:
                 parse_failures.append(renewal_file)
+            elif lineage_config.certname  and cli.set_by_cli('autorenew'):
```

Comment:
Let's move the overall check into `storage.should_autorenew`.

Reply:
Moved the check to `certbot.storage.RenewableCert.should_autorenew`.